erpnext/patches/v11_1/woocommerce_set_creation_user.py

```diff
@@ -4,7 +4,13 @@
 def execute():
 	woocommerce_setting_enable_sync = frappe.db.sql("SELECT t.value FROM tabSingles t WHERE doctype = 'Woocommerce Settings' AND field = 'enable_sync'",  as_dict=True)
 	if len(woocommerce_setting_enable_sync) and woocommerce_setting_enable_sync[0].value == '1':
-		frappe.db.sql("""UPDATE tabSingles
-					SET value = (SELECT t.value FROM tabSingles t WHERE doctype = 'Woocommerce Settings' AND field = 'modified_by')
-					WHERE doctype = 'Woocommerce Settings'
-					AND field = 'creation_user';""")
\ No newline at end of file
+		modified_by = frappe.db.sql("""SELECT t.value
```

Comment:
please no hanging indents

https://github.com/frappe/erpnext/wiki/Coding-Standards#use-simple-indenting

Reply:
it does look strange on github... below is how it looked on vscode:
![hanging_SQL](https://user-images.githubusercontent.com/4847317/56292119-2fda0b00-6144-11e9-81fe-f9687c423892.png)

I have updated it now...